frontend/index.html

```diff
@@ -3,10 +3,8 @@
   <head>
     <meta charset="UTF-8" />
     <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
-    <link rel="icon" type="image/png" sizes="32x32" href="./public/favicon_package_v0.16/favicon-32x32.png" />
-    <link rel="icon" type="image/png" sizes="16x16" href="./public/favicon_package_v0.16/favicon-16x16.png" />
-    <link rel="manifest" href="/site.webmanifest" />
-    <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5" />
+    <link rel="icon" type="image/png" sizes="32x32" href="/favicon_package_v0.16/favicon-32x32.png" />
+    <link rel="icon" type="image/png" sizes="16x16" href="/favicon_package_v0.16/favicon-16x16.png" />
```

Comment:
These were added when we added favicon, deleted redundant ones and updated paths